app/src/main/java/org/mozilla/fenix/components/metrics/LeanplumMetricsService.kt

```diff
@@ -34,7 +34,9 @@ private val Event.name: String?
         is Event.InteractWithSearchURLArea -> "E_Interact_With_Search_URL_Area"
         is Event.CollectionSaved -> "E_Collection_Created"
         is Event.CollectionTabRestored -> "E_Collection_Tab_Opened"
-        is Event.SyncAuthSignIn -> "E_Sign_In_FxA"
+        is Event.SyncAuthSignUp -> "E_FxA_New_Signup"
+        is Event.SyncAuthSignIn, Event.SyncAuthPaired,
+        Event.SyncAuthFromShared, Event.SyncAuthOtherExternal -> "E_Sign_In_FxA"
```

Comment:
`SyncAuthFromShared` should be split into a separate event. This is how migrated users will show up, and we don't want to track this as a user-action since that's just part of an automated process. E.g. user didn't "sign-in" in that case, they were already signed-in, we simply upgraded their app.

We'll have a long tail of users migrating even after the rollout is complete, so unless this is split it'll skew our sign-in metrics.

Reply:
Some more nuance here: this is _also_ how the users who pressed "auto-sign-in" button in the onboarding UI will show up, but now that Fennec is gone there's currently not a auth provider present (fenix can currently only consume credentials, not emit them to other moz apps).